tests/test_render_pass.py

```diff
@@ -0,0 +1,127 @@
+from pyvista import _vtk
+from pyvista.plotting.render_passes import RenderPasses
+
+
+# this ideally would be a fixture, but if it's a fixture the renderer object
+# collects immediately since RenderPasses only holds a weakref
+def make_passes():
+    ren = _vtk.vtkRenderer()
+    passes = RenderPasses(ren)
+    return ren, passes
```

Comment:
OK, I understand the updated comment then, thanks. So how about returning both the renderer and the passes from the fixture? It would all look the same, just called `passes` (or something like `renderer_with_passes`) instead of `make_passes`. The only downside is that you need to unpack or index into the fixture when you use it. And you also have to delete the name of the fixture (the tuple) in tests where you want to check collection.

Reply:
I originally went with that approach but I felt it wasn't I. The spirit of fixtures and decided against it.